11ty/definitions/spiritual-bypassing.md

```diff
@@ -0,0 +1,48 @@
+
+## Issues
+
+Spiritual bypassing avoids deeply addressing discomfort by offering surface-level platitudes, often in the form of co-opted religious, philosophical, and spiritual practices, terminology, and imagery. 
+
+Spiritual bypassing is also referred to as toxic positivity. It is commonly found in wellness spaces, temples, churches, hospitals, therapy centers, and self-help groups.
+
+Wellness instructors often perpetuate spiritual bypassing by modeling the behavior they see from others practitioners, notably on social media. They are oftentimes unaware of its negative aspects and view the behavior as a deeper investment in the practice.
+
+Furthermore, co-opted language and imagery is often drawn specifically from tenets of faith. These concepts are often presented by people who do not identify with the underlying belief system. They are also often presented without attribution or context, and the instance of application may be in conflict with the original cultural context.
+
+## Examples
+
+Saying a phrase such as, “we are all one” or “we are all human” when confronted with difficulty or hardship.
+
+## Impact
+
+Not addressing discomfort means an individual will not process and internalize difficult or painful events. Escapism of this form prevents the individual from working through hardship. 
+
+Failing to acknowledge hardship is especially problematic in the context of minoritized groups. Downplaying invalidates both their lived experiences and creates an environment that both disincentives further participation and suppresses trauma.
+
+[Other possible effects of spiritual bypassing](https://www.researchgate.net/publication/319321341_The_Opiate_of_the_Masses_Measuring_Spiritual_Bypass_and_Its_Relationship_to_Spirituality_Religion_Mindfulness_Psychological_Distress_and_Personality) include dichotomous thinking, obsession, codependence, narcissism, spiritual materialism, and abdication of personal responsibility.
+
+Also understand the chilling effects it has on minoritized communities, and how it affects their presence (or lack of presence) in spaces where spiritual bypassing is practiced.
+
+## What to do instead
+
+Understanding how to identify spiritual bypassing as a discrete behavior and any co-opted concepts that are used to perform it is an important first step. It can be present both in-person and on social media channels.
+
+Identification means an individual can cease these behaviors, if practicing them. If witnessed, it is advised to use indirect questioning techniques such as [motivational interviewing](https://en.wikipedia.org/wiki/Motivational_interviewing) to ascertain the underlying factors compelling this behavior. Direct confrontation may have opposite of the desired effect. It may be construed as an attack on the person's authority, or cause them to double-down and further incorporate it into their sense of identity.
```

Comment:
```suggestion
Identification means an individual can cease these behaviors, if practicing them. If witnessed, it is advised to use indirect questioning techniques such as [motivational interviewing](https://en.wikipedia.org/wiki/Motivational_interviewing) to ascertain the underlying factors compelling this behaviour.

Direct confrontation may have opposite of the desired effect. It may be construed as an attack on the person's authority, or cause them to double-down and further incorporate this behaviour into their sense of identity and cause even more harm.
```